app/jobs/in_person/ipp_locations_cacher_job.rb

```diff
@@ -0,0 +1,21 @@
+module InPerson
+  class IppLocationsCacherJob < ApplicationJob
+    include ArcgisApi
+
+    def perform(post_offices)
+      post_offices.each do |location|
+        location = Geocoder.new.find_address_candidates(
+          Address: location[:address],
+          City: location[:city],
+          Region: location[:state],
+          Postal: location[:zip_code_5],
+        ).first.location
+
+        UspsIppCachedLocations.create!(
+          lonlat: RGeo::Geos.factory(srid: 4326).point(location.longitude, location.latitude),
```

Comment:
I had to look up what this number was, I'd recommend pulling it into a constnat so we can name it 
```suggestion
          lonlat: RGeo::Geos.factory(srid: COORDINATES_WGS84).point(location.longitude, location.latitude),
```

```ruby
COORDINATES_WGS84 = 4326
```